bot.py

```diff
@@ -124,21 +125,25 @@ async def on_message_delete(message):
         del starboard_messages[message.id]
 
 
+def is_trivial_command(c: str) -> Optional[str]:
+    cmd = c.strip()
+    if not cmd.startswith("?"):
+        return None
+    if cmd.count(" "):
+        return None
+
+    return cmds.get(cmd, None)
+
+
 @client.event
 async def on_message(message: discord.Message):
     if message.author.bot:
         return
-    if message.content in cmds and message.content != "//":
-        # if string return string, if list return random element
-        if type(cmds[message.content]) == list:
-            response = random.choice(cmds[message.content])
-        else:
-            response = cmds[message.content]
-        await message.channel.send(
-            SHERP_URL + response if message.author.id == SHERP_ID else response
-        )
+
+    if resp := is_trivial_command(message.content):
+        await message.channel.send(resp)
         return
-    # find message.content in commands.json and append msg with the value
+
     elif "?pasta" in message.content:
```

Comment:
noticed that this will trigger if the msg doesn't start with ?pasta (`foo?pastabar`). can it be fixed in this change?

Reply:
This is not part of this is PR. ?pasta is not a trivial command since it requires logic (choosing a random one). It will be addressed in a future PR.

Reply:
The code for pasta remains unchanged from what it was originally.